File: articles/contestability-appeals-and-algorithmic-due-process/go/contestability_score.go
package main

import "fmt"

func main() {
	notice := 0.70
	reasons := 0.62
	evidenceAccess := 0.48
	humanReview := 0.55
	correction := 0.52
	remedy := 0.44
	score := (notice + reasons + evidenceAccess + humanReview + correction + remedy) / 6.0
	fmt.Printf("contestability_score=%.4f\n", score)
}

File: articles/contestability-appeals-and-algorithmic-due-process/cpp/contestability_score.cpp
#include <iostream>

int main() {
    double notice = 0.70;
    double reasons = 0.62;
    double evidence_access = 0.48;
    double human_review = 0.55;
    double correction = 0.52;
    double remedy = 0.44;
    double score = (notice + reasons + evidence_access + human_review + correction + remedy) / 6.0;
    std::cout << "contestability_score=" << score << "\n";
    return 0;
}
